### PR TITLE
AO3-5745 AO3-5856 Add CSV link for prompt memes, add Title field to CSV

### PR DESCRIPTION
**Pull request by sarken**
## Issue

https://otwarchive.atlassian.net/browse/AO3-5745
https://otwarchive.atlassian.net/browse/AO3-5856

## Purpose

* Adds a CSV download link for prompt meme maintainers
* Removes email from prompt meme CSVs
* Adds Title field to gift exchange and prompt meme CSVs (honestly, I don't think this is important, so I'm happy to take it out of this pull request)

## Testing Instructions

Refer to Jira.

### CODE REVIEW

**Review by brianjaustin — APPROVED** [review]
LGTM!

File: app/controllers/challenge_signups_controller.rb [hunk 1]
```diff
@@ -330,7 +337,6 @@ def prompt_meme_to_csv
           ["(Anonymous)", "", ""]
```
[the remark this answers is not in the document]
Comment (sarken) [reply app/controllers/challenge_signups_controller.rb:337]: Yeah, you're right -- I didn't test with anonymous prompts.

File: app/views/challenge/shared/_challenge_requests.html.erb [hunk 2]
```diff
@@ -4,6 +4,12 @@
 
 <!--subnav-->
 <ul class="navigation actions" role="navigation">
+  <% if @collection.user_is_maintainer?(current_user) %>
+    <li>
+      <%= link_to ts("Download (CSV)"), collection_signups_path(@collection, format: :csv) %>
+      <%= link_to_help "csv-download"%>
```
Comment (tickinginstant) [inline app/views/challenge/shared/_challenge_requests.html.erb:10]: Can you put a space before the closing `%>` here?

**Review by tickinginstant — COMMENTED** [review]
Sorry, noticed one last thing while I was checking over the rest of the code. But other than that, this should be good to go.

**Review by tickinginstant — APPROVED** [review]
Looks good, thanks!

**Pull request merged**